docs/pages/FileSystem.md

```diff
@@ -0,0 +1,38 @@
+# UI5 FS
+
+The [UI5 FS](https://github.com/SAP/ui5-fs) provides a UI5-specific file system abstraction.
+
+[**API Reference**](https://sap.github.io/ui5-tooling/v3/api/){: .md-button .sap-icon-initiative }
+
+## Overview
+
+The virtual file system "UI5 FS" offers an abstraction layer from the physical file system. Among other features, it can combine a set of scattered file locations into a well defined, virtual structure.
```

Comment:
```suggestion
The virtual file system "UI5 FS" offers an abstraction layer from the physical file system. Among other features, it can combine a set of scattered file locations into a well-defined virtual structure.
```